app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -19,28 +19,32 @@
           class="select-role">
           <ui-select-match placeholder="Select a storage class">
             <span>
-              {{$select.selected.metadata.name}}
+              {{$select.selected.metadata.name}}<br/>
             </span>
           </ui-select-match>
           <ui-select-choices
             repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } ">
             <div>
               <span ng-bind-html="sclass.metadata.name  | highlight : $select.search"></span>
-              <span ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description)" class="text-muted">
-                <small>&ndash;
+              <div ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description) || (sclass | annotation : 'storage.alpha.openshift.io/access-mode')" class="text-muted">
+                <small>&nbsp;
```

Comment:
Why is `&nbsp;` needed?

Reply:
I thought it looked better:
With blank -
![withblank](https://user-images.githubusercontent.com/18728857/36397642-d636afc8-1580-11e8-9fa8-8e1a5860c4dc.png)

Without Blank -
![withoutblank](https://user-images.githubusercontent.com/18728857/36397650-ddfc8d90-1580-11e8-9f53-79653d1b3ca4.png)

What do you think?

Reply:
Left-aligned is consistent with other similar messages, so I'd leave out the leading space. If we really want that, we should use margins instead anyway. But I like it better without.